frontend/src/mock-data/TodoTaskList/TodotaskListMock.ts

```diff
@@ -0,0 +1,61 @@
+import { TodoTaskListProps } from '../../components/TodoTaskList/types';
+
+export const ListMock: TodoTaskListProps = {
```

Comment:
zamiast tworzyć `TodoTaskListProps` i importować ten typ tutaj to można zrobić po prostu `ListMock: TodoTaskProps[]`

Reply:
Nie może już zostać jak jest? Musiałbym sporo przerabiać.
![image](https://user-images.githubusercontent.com/61019376/167527167-0a22ab8e-50db-4f1c-a944-e7f52214843c.png)